form/form_for.go

```diff
@@ -137,8 +137,14 @@ func (f FormFor) value(field string) interface{} {
 	}
 
 	i := fn.Interface()
-	if dv, ok := i.(nulls.String); ok {
-		return dv.String
+	if dv, ok := i.(driver.Valuer); ok {
+		value, _ := dv.Value()
+
+		if value == nil {
+			return ""
+		}
+
+		return fmt.Sprintf("%v", value)
```

Comment:
no need to convert this to a string, `dv.Value()` returns an `interface{}`, and this function returns an `interface{}` so we should be ok just returning that.